spec/forms/decidim/action_delegator/admin/delegation_form_spec.rb

```diff
@@ -5,8 +5,7 @@
 describe Decidim::ActionDelegator::Admin::DelegationForm do
   subject { described_class.from_params(attributes) }
 
-  let(:attributes) { { granter_id: granter.id, grantee_id: grantee.id, setting: current_setting } }
-  let(:current_setting) { create(:setting) }
+  let(:attributes) { { granter_id: granter.id, grantee_id: grantee.id } }
```

Comment:
it's probably slightly more readable and changeable if we move this let to the first context, where it's used only.